moderation.go

```diff
@@ -42,3 +42,68 @@ func (c *Client) GetBannedUsers(params *BannedUsersParams) (*BannedUsersResponse
 
 	return bans, nil
 }
+
+type BanUserParams struct {
+	BroadcasterID string             `json:"broadcaster_id"`
+	ModeratorId   string             `json:"moderator_id"`
+	Body          BanUserRequestBody `json:"data"`
+}
+
+type BanUserRequestBody struct {
+	Duration int    `json:"duration"` // optional
+	Reason   string `json:"reason"`   // required
+	UserId   string `json:"user_id"`  // required
+}
+
+type BanUserResponse struct {
+	ResponseCommon
+	Data ManyBanUser
+}
+
+type ManyBanUser struct {
+	Bans []BanUser `json:"data"`
+}
+
+type BanUser struct {
+	BoardcasterId string `json:"broadcaster_id"`
+	CreatedAt     string `json:"created_at"`
+	EndTime       string `json:"end_time"`
```

Comment:
Both `CreatedAt` and `EndTime` fields should be of type [Time](https://github.com/nicklaw5/helix/blob/main/time.go#L17-L19). This allows the our custom [UnmarshalJSON](https://github.com/nicklaw5/helix/blob/main/time.go#L26-L37) to handle converting the string represented time to a native [time.Time](https://pkg.go.dev/time#Time) type.

See the [User type](https://github.com/nicklaw5/helix/blob/main/users.go#L16) as an example.